server/messages/messageController.js

```diff
@@ -1,45 +1,71 @@
 'use strict';
 // built-in npm module dependencies
-var Promise = require('bluebird');
+//var Promise = require('bluebird');
+//var Promise = require('mpromise');
+var _ = require('underscore');
 
 // local dependencies
 var Message = require('./messageModel.js');
 
 module.exports = {
   addMessage: function (request, response) {
-  // // logic to add or update Message after server receives POST request
-  //   // convert phone number to number
-  //   var messageObject = {
-  //     userId: request.body.user,
-  //     // may have to convert the contactId to an ObjectId to display the relationship
-  //     // see: mongoosejs.com/docs/populate.html
-  //     contactId: request.body.contactId,
-  //     contactPhone: request.body.phone,
-  //     text: request.body.message,
-  //     date: request.body.date,
-  //     status: 'scheduled'
-  //   };
-  //   var newMessage = new Message(messageObject);
-  //   newMessage.save(function (error) {
-  //     if (error) { console.log(error); }
-  //   });
+  // logic to add or update Message after server receives POST request
+    var messageObject = {
+      userId: request.body.userId,
+      contactId: request.body.contactId,
+      text: request.body.text,
+      date: new Date(),
```

Comment:
@onelovelyname we'll need to remember to remove the hardcoding of `new Date()` and replace it with the proper code for setting the date/time entered by the user